field/src/field.rs

```diff
@@ -190,6 +190,7 @@ pub trait PrimeField: Field + Ord {}
 pub trait PrimeField64: PrimeField {
     const ORDER_U64: u64;
 
+    // TODO: Move to Field itself? Limiting it to `PrimeField64` seems unusual.
```

Comment:
Hm that makes sense — we'd just need each Field to keep track of its size as a BigUint I think

Reply:
I was thinking maybe `fn bits();` with no default impl so that we don't need any dependencies, though maybe a dependency on `num_biguint` is alright